### PR TITLE
Improve GUI for better user experince

### PR DESCRIPTION
**Pull request by alice221927**
Changes in a nutshell:
- Added Notification Banner with timer. This Notification appears when program starts to inform the user about user manual
- Added help menu with user manual action
-added tooltips
- changed the name of view options to set thresholds

What is missing?
The user manual. I am not sure about the format. Pdf? Txt? 
When the user clicks on user manual, the user manual opens. For that There are two options:
- File get opened via the controller (new function needed)
- File get opened directly from the view (violates the role of view and controller)
What is your opinion on that?

### CODE REVIEW

**Comment by alice221927** [comment]
Please consider that the user manual is an important part of our program, because it improves the user's experience a lot and prevents most issues, which the users faced during the user tests.

File: src/view/ui/evaluation.ui [hunk 1]
```diff
@@ -28,7 +37,7 @@
      <item>
       <widget class="QToolButton" name="view_options_button">
        <property name="text">
-        <string>View Options</string>
+        <string>Set Thresholds</string>
```
[the remark this answers is not in the document]
Comment (alice221927) [reply src/view/ui/evaluation.ui:40]: No the thresholds cannot be removed completely. The user can set them all to 0..

[the remark this answers is not in the document]
Comment (alice221927) [reply src/view/ui/evaluation.ui:40]: What about my questions in the description? Do you think the view should do the entire job or is it the controller responsibility? @Mikrischu

[the remark this answers is not in the document]
Comment (alice221927) [reply src/view/ui/evaluation.ui:40]: @Mikrischu 
Do I need a new Controller/Manager for that or can I just implement it in FileManager?

**Comment by alice221927** [comment]
@Mikrischu 
@Kevin2471 
@FliFlaFloo 
@einDamian 

I added a new function to the project manager. is that fine?

File: src/controller/ProjectManager.py [hunk 2]
```diff
@@ -364,3 +372,8 @@ def import_raw_data(self, path: str):
         """
         raw_data = DataFrame(FileManager.import_(path))
         self.get_project().set_raw_data(raw_data, path)
+
+    @staticmethod
+    def open_user_manual():
+        path = os.path.join(Path(__file__).parent.parent, "user_manual.pdf")
+        subprocess.Popen([path], shell=True)
```
Comment (FliFlaFloo) [inline src/controller/ProjectManager.py:379]: Not sure if that's my problem, but the user manual does not open for me, I get the following error: 
/bin/sh: /Users/flo/Documents/Kit/discrete-choice/src/user_manual.pdf: Permission denied
just fyi

Comment (alice221927) [reply src/controller/ProjectManager.py:379]: I think this is OS dependent. It's working fine for me.

**Pull request merged**